commit 80570a89f0736b8f0c8d5371f2331ddeba9b3bef
Author: q66 <q66@chimera-linux.org>
Date:   Sun Jan 5 11:44:11 2025 +0100

    df: treat ENOTCONN as a reason to skip a filesystem
    
    It seems this is returned for FUSE filesystems when they are dead.

diff --git a/src.freebsd/coreutils/df/df.c b/src.freebsd/coreutils/df/df.c
index 064fccd..51444bc 100644
--- a/src.freebsd/coreutils/df/df.c
+++ b/src.freebsd/coreutils/df/df.c
@@ -691,7 +691,7 @@ getmntinfo(struct mntinfo **mntbuf)
 
 	    /* get stat(vfs) fields and copy those over */
 	    if (statvfs(ent->mnt_dir, &svfsbuf) == -1 || stat(ent->mnt_dir, &stmnt) == -1) {
-	        if ((errno == EACCES) || (errno == EPERM)) continue;
+	        if ((errno == EACCES) || (errno == EPERM) || (errno == ENOTCONN)) continue;
 	        xo_err(1, "statvfs");
 	    }
 
